namer/dnssrv/src/test/scala/io/buoyant/namer/dnssrv/DnsSrvNamerTest.scala

```diff
@@ -0,0 +1,38 @@
+package io.buoyant.namer.dnssrv
+
+import com.twitter.finagle._
+import com.twitter.finagle.util.LoadService
+import io.buoyant.config.Parser
+import io.buoyant.namer.{NamerConfig, NamerInitializer}
+import io.buoyant.test.FunSuite
+import org.scalatest.Matchers
+
+class DnsSrvNamerTest extends FunSuite with Matchers {
+
+  test("sanity") {
+    // ensure it doesn't totally blowup
+    val _ = DnsSrvNamerConfig(
+      Some(15),
+      Some(List("localhost"))
+    ).newNamer(Stack.Params.empty)
+  }
+
+  test("service registration") {
+    assert(LoadService[NamerInitializer]().exists(_.isInstanceOf[DnsSrvNamerInitializer]))
+  }
+
+  test("parse config") {
+    val yaml = s"""
+                  |kind: io.l5d.dnssrv
+                  |experimental: true
```

Comment:
should I make this required? If so, where?

Reply:
Set `experimentalRequired = true` in `DnsSrvNamerInitializer`. See (for example) `K8sNamerInitializer`:
https://github.com/linkerd/linkerd/blob/5d22b56489c4e646672e5d7fce6ef799d7574397/namer/k8s/src/main/scala/io/buoyant/namer/k8s/K8sExternalInitializer.scala#L34-L35